src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -283,6 +284,7 @@ namespace ngraph
             REGISTER_OPERATOR("Pow", 1, pow);
             REGISTER_OPERATOR("PRelu", 1, prelu);
             REGISTER_OPERATOR("QLinearConv", 1, quant_conv);
+            REGISTER_OPERATOR("QLinearMatMul", 1, qmatmul);
```

Comment:
Please preserve style: the other quantized ops functions are named with `quant` or `quantize` (_full_) prefix.